src/stores/ApiController.ts

```diff
@@ -0,0 +1,52 @@
+import { injectable } from 'inversify';
+
+import type { GetPlaylistParams, Playlist, PlaylistItem } from '#types/playlist';
+import type { AdSchedule } from '#types/ad-schedule';
+import type { EpisodeInSeries, EpisodesWithPagination, GetSeriesParams, Series } from '#types/series';
+import ApiService from '#src/services/api.service';
+
+@injectable()
+export default class ApiController {
```

Comment:
I really think it's in our long term interest to avoid creating controllers that are mostly or entirely passthrough into services. It's just a lot of extra code to maintain and interfaces to ensure we keep matched up. I would argue that hooks are equivalent to controllers so should be allowed to call services.

Containers or components calling services is a little more grey, but maybe those ones we can handle case by case.

Reply:
Removing duplicates